disco_aws_automation/disco_aws.py

```diff
@@ -409,6 +427,11 @@ def provision(self, ami, hostclass=None,
 
         self.create_scaling_schedule(hostclass, min_size, desired_size, max_size)
 
+        # Create alarms and custom metrics for the hostclass, if is not being used for testing
+        if not testing:
+            hostclass_alarms = self.alarm_configs.get_alarms(hostclass, group.name)
+            self.alarms.create_alarms(hostclass_alarms)
```

Comment:
Which logic? Not creating alarms if we're in testing mode? Or have `self.alarms.create_alarms()` accept hostclass and group name and generate the configs itself?